.github/workflows/main.yml

```diff
@@ -35,9 +35,10 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r etc/${{matrix.requirements_file}}
         pip install -e .
-    - name: Lint with flake8
-      run: |
-        flake8
```

Comment:
Ruff is run as part of pre-commit, so it fails if linting fails => no need for a dedicated step in here any more.